File: Models/Reply.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Models
{
    public partial class Reply
    {
        public int ReplyId { get; set; }
        public string ReplyMessage { get; set; }
        public DateTime? ReplyTimestamp { get; set; }
        public int? ProfileId { get; set; }
        public int? TopicId { get; set; }

        public virtual Profile Profile { get; set; }
        public virtual Topic Topic { get; set; }
    }
}

File: pawpers-angular/src/app/AngularModels/reply.ts
export interface Main {
    $id:     string;
    $values: Reply[];
}

export interface Reply {
    $id:            string;
    replyId:        number;
    replyMessage:   string;
    replyTimestamp: null;
    profileId:      number;
    topicId:        number;
    profile:        null;
    topic:          null;
}
